README.md

```diff
@@ -217,6 +217,13 @@ for more details on the structure of the metadata file.
 
   If `true`, forces a re-upload of releases of releases and versions that are already present on Tanzu Network.
 
+* `files_only`: *Optional boolean.*
```

Comment:
Is this boolean used? We ignored it when testing on our pipelines and looking at the logic it seems we only use ExistingRelease from the metadata.

Reply:
Excellent question -- i'll run the tests without it and remove it if it's not necessary, and update the docs :)